src/test/scala/cromwell/webservice/CromwellApiServiceIntegrationSpec.scala

```diff
@@ -11,8 +12,8 @@ import spray.json._
 import spray.testkit.ScalatestRouteTest
 
 class CromwellApiServiceIntegrationSpec extends FlatSpec with CromwellApiService with ScalatestRouteTest with Matchers {
-  def actorRefFactory = system
-  val workflowManager = TestActorRef(new WorkflowManagerActor(new LocalBackend))
+  def actorRefFactory = new TestWorkflowManagerSystem().actorSystem
+  val workflowManager = TestActorRef(new WorkflowManagerActor(new LocalBackend(system)))
```

Comment:
I think I'm confused about something... in other places you did `LocalBackend(new CromwellTestkitSpec.TestWorkflowManagerSystem().actorSystem)` and here you're doing `new LocalBackend(system)`... should this be `new LocalBackend(actorRefFactory)`?